### PR TITLE
added random seed option for double well

### PR DESCRIPTION
**Pull request by brookehus**
I decided it was easier to add a parameter to `init` but let me know if you prefer something different. What kind of tests do you want for this?

- [ ] Make sure to include one or more tests for your change
- [x] Add yourself to `AUTHORS`
- [x] Add a new entry to the `doc/source/CHANGELOG` (choose any open position to avoid merge conflicts with other PRs).
      Decide whether your change is a fix or a new feature.

### CODE REVIEW

**Comment by brookehus** [comment]
cc @thempel

File: pyemma/datasets/double_well_thermo.py [hunk 1]
```diff
@@ -39,6 +39,12 @@ def __init__(self):
         self.pi[:] = self.pi / self.pi.sum()
         self.f = -_np.log(self.pi)
 
+        if random_seed is not None:
+            self._rng = _np.random.RandomState(random_seed)
```
Comment (marscher) [inline pyemma/datasets/double_well_thermo.py:43]: does this overwrite the state for all numpy users? If that is the case, is there a way to circumvent reseeding others random state?

Comment (brookehus) [reply pyemma/datasets/double_well_thermo.py:43]: No, I don't think so. I tested:

```
outer_rng = np.random.RandomState(100)
outer_rng.randn(1)
dw_sampler = DoubleWellSampler(random_seed=1)
outer_rng.randn(1)
p1 = dw_sampler.sample(length=10)['trajs'][0]
print(np.random.randn(1))
print(outer_rng.randn(1))
```

And the `outer_rng` is unaffected by the `random_seed` argument to `DoubleWellSampler` nor its execution, and the `np.random` call is different on different runs with the same `outer_rng` and `random_seed`.

Comment (brookehus) [reply pyemma/datasets/double_well_thermo.py:43]: As far as I know the random state is only for calls that come from `_rng` in place of `_np.random`.

Comment (brookehus) [reply pyemma/datasets/double_well_thermo.py:43]: One consideration though is that if the same `DoubleWellSampler` object were used to run `sample` more than once without a new instance of the `Sampler`, then each result would be different.

Comment (marscher) [reply pyemma/datasets/double_well_thermo.py:43]: Thanks for testing this. Why would this be an issue in the latter case? I would expect it to be different.

Comment (brookehus) [reply pyemma/datasets/double_well_thermo.py:43]: Yeah, same. Not an issue.

**Comment by marscher** [comment]
Should I wait for a test or not? This seems like a minor change, which is not really exposed to user api.

**Comment by brookehus** [comment]
If there should be a test let me know what I should test. I don't mind either way.

**Comment by marscher** [comment]
LGTM

**Pull request merged**